integration-tests/testutils/common/common.go

```diff
@@ -39,8 +39,9 @@ import (
 const (
 	// BaseAltPartitionPath is the path to the B system partition.
 	BaseAltPartitionPath = "/writable/cache/system"
-	needsRebootFile      = "/tmp/needs-reboot"
-	channelCfgFile       = "/etc/system-image/channel.ini"
+	// NeedsRebootFile is the file that a test writes in order to request a reboot.
+	NeedsRebootFile = "/tmp/needs-reboot"
```

Comment:
If this is only exported so that it can be used in tests, we could use the pattern that Gustavo was suggesting for these kinds of situations. The idea is to have a "export_test.go" file that contains a line like:
```
// expose test-only things here
var NeedsRebootFile = needsRebootFile
```

@niemeyer please correct me here if I misrepresented this pattern